bin/lib/cli/environment.py

```diff
@@ -43,14 +46,17 @@ def environment_start(cfg: Config):
               help='While updating, ensure at least PERCENT are healthy', default=75, show_default=True)
 @click.option('--motd', type=str, default='Site is being updated',
               help='Set the message of the day used during refresh', show_default=True)
+@click.option('--no-notify', type=bool)
```

Comment:
I vaguely remember seeing something that click can do `--notify/no-notify`? Or am I imagining things?

Reply:
Yes! https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags